2024-summer-FE-seminar/packages/web/src/features/my/user/_mock/chachaMockUpPhone.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiUsr001ResponseOK } from "@sparcs-clubs/interface/api/user/endpoint/apiUsr001";
+
+const chachaMockUpMyPhone: ApiUsr001ResponseOk = {
+  clubs: [
+    {
+      id: 1,
+      name: "술박스",
+    },
+    {
+      id: 2,
+      name: "술술박스",
+    },
+  ],
+  name: "안차차",
+  email: "chayunahn@kaist.ac.kr",
+  department: "전산학부",
+  studentNumber: 20220390,
+  phoneNumber: "01071823758",
+};
+
+export default { chachaMockUpMyPhone };
```

Comment:
이거를 이렇게 바꾸면 chachaMockUpMyPhone.phoneNumber로 값 가져올 수 있어요
```suggestion
export default chachaMockUpMyPhone;
```